src/components/FeeExplainer/FeeSliders.tsx

```diff
@@ -47,18 +59,50 @@ export const FeeSliders: React.FC<FeeSlidersProps> = ({
   const {
     number: { toFiat },
   } = useLocaleFormatter()
+
+  const handleSetFoxHolding = useCallback(
+    (values: NumberFormatValues) => {
+      setFoxHolding(bnOrZero(values.value).toNumber())
+    },
+    [setFoxHolding],
+  )
+
+  const debounceSetFoxHolding = useMemo(
+    () => debounce(handleSetFoxHolding, 1000),
+    [handleSetFoxHolding],
+  )
+
+  const handleSetTradeSizeUsd = useCallback(
+    (values: NumberFormatValues) => {
+      setTradeSizeUSD(bnOrZero(values.value).toNumber())
+    },
+    [setTradeSizeUSD],
+  )
+
+  const debounceSetTradeSizeUsd = useMemo(
+    () => debounce(handleSetTradeSizeUsd, 1000),
+    [handleSetTradeSizeUsd],
+  )
+
   return (
     <VStack height='100%' spacing={8} mt={6}>
       <Stack spacing={4} width='full'>
-        <Flex width='full' justifyContent='space-between' fontWeight='medium'>
+        <Flex width='full' justifyContent='space-between' alignItems='center' fontWeight='medium'>
           <Text translation='foxDiscounts.foxPower' />
-          <Skeleton isLoaded={!isLoading}>
-            <Amount.Crypto
-              value={foxHolding.toString()}
-              symbol='FOX'
-              fontWeight='bold'
-              maximumFractionDigits={0}
-            />
+          <Skeleton isLoaded={!isLoading} width='35%'>
+            <Box sx={inputStyle}>
+              <NumberFormat
+                decimalScale={2}
+                customInput={Input}
+                isNumericString={true}
+                suffix={' FOX'}
+                decimalSeparator={'.'}
+                inputMode='decimal'
+                thousandSeparator={','}
```

Comment:
@NeOMakinG any reason why this is hardcoded vs. using `localeParts.decimal` and `localeParts.group`?

Reply:
Changed in [6b12ae3](https://github.com/shapeshift/web/pull/7984/commits/6b12ae3b57f16c7a37fc2699a84daf2c62b3147f)